millw

```diff
@@ -92,6 +92,37 @@ fi
 
 MILL="${MILL_DOWNLOAD_PATH}/${MILL_VERSION}"
 
+try_to_use_system_mill() {
+  MILL_IN_PATH=$(command -v mill)
+
+  if [ -z "${MILL_IN_PATH}" ]; then
+    return
+  fi
+
+  UNIVERSAL_SCRIPT_MAGIC="@ 2>/dev/null # 2>nul & echo off & goto BOF"
+
+  if ! head -c 128 "${MILL_IN_PATH}" | grep -qF "${UNIVERSAL_SCRIPT_MAGIC}"; then
+    echo "Could not determine mill version as ${MILL_IN_PATH} does not start with the universal script magic2"
```

Comment:
Messages from the wrapper should go to STDERR.

```suggestion
    echo "Could not determine mill version as ${MILL_IN_PATH} does not start with the universal script magic2" 1>&2
```

Reply:
I am torn with regard to the messages the mill wrapper emits. I actually also would consider removing them altogether. They are not really error messages, maybe not even really warning messages. I leave it up to you to decide if I should remove them or redirect them to stderr.

Reply:
It's not whether these are error but to which channel they should print. As a mill wrapper, we won't interfere with any output a user might expect from mill, so we have to use STDERR.